src/components/payments/payments.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PaymentsList from '../payments-list/paymentsList'
+
+const PaymentsPage = ({users, spendings}) => {
+    let style = '';
+
+    if (users.length <= 1) {
```

Comment:
I'd moved this logic to the parent component, so that we do not render the <PaymentsPage> if there is less than 2 users available.

Something like

```javascript
   {users.length > 1 && <PaymentsPage users={users} spendings={spendings} />}
```